mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/impl/MessageParserTest.java

```diff
@@ -63,6 +63,15 @@ public void getAttachmentsShouldRetrieveAttachmentNameWhenOne() throws Exception
     }
 
     @Test
+    public void getAttachmentsShouldRetrieveAttachmentNameWhenOneContainingNonASCIICharacters() throws Exception {
+        List<MessageAttachment> attachments = testee.retrieveAttachments(ClassLoader.getSystemResourceAsStream("eml/messageWithNonASCIIFilenameAttachment.eml"));
+
+        assertThat(attachments).hasSize(1);
+        Optional<String> expectedName = Optional.of("لاتتطور أم.png");
```

Comment:
Maybe you can use some text file that is not even a few byte long rather than a 119 KB png, as you are only interested in the file name ;-)